estate/models/estate_property.py

```diff
@@ -0,0 +1,21 @@
+from odoo import fields, models
+
+class EstateModel(models.Model):
+    _name = "estate_property"
+    _description = "Estate Property"
+
+    name = fields.Char(required=True, default="New House")
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False, default=fields.Date.add(fields.Date.today(), months=3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(readonly=True, copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    garage = fields.Boolean()
+    garden = fields.Boolean()
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(selection=[("north", "North"), ("south", "South"), ("east", "East"), ("west", "West")])
+    state = fields.Selection(selection=[("new", "New"), ("offer_received", "Offer Received"), ("offer_accepted", "Offer Accepted"), ("sold", "Sold"), ("cancelles", "Cancelled")], default="new", required=True)
```

Comment:
```suggestion
    state = fields.Selection(
        selection=[
            ("new", "New"), 
            ("offer_received", "Offer Received"), 
            ("offer_accepted", "Offer Accepted"), 
            ("sold", "Sold"), 
            ("cancelled", "Cancelled")
        ], 
        default="new", 
        required=True)
```
Don't hesitate to give space when you can. You can get inspired from the code you find in Odoo in general.  This comment also applies to other fields that are too long. 
Be careful it was written cancelles instead of cancelled.
You can also give a string attribute to give a proper name in the interface